### PR TITLE
Podcasts rebranding

### PR DESCRIPTION
**Pull request by ewels**
* Add new podcast logo to the top of the page
* Update podcast cover art for previous episodes

### CODE REVIEW

File: templates/podcastsindex.ftl [hunk 1]
```diff
@@ -7,15 +7,22 @@
...
       <div class="col-xs-12">
-        <h4 class='text-muted'>Talking about news in the Nextflow ecosytem and speaking with pioneers in the field.</h4>
+        <div class="row">
+          <div class="col-sm-4">
+            <img src="/img/podcast_logo_horiz.svg">
+          </div>
+          <div class="col-sm-8">
+            <h4 class='text-muted' style="margin-top: 6rem;">Talking about news in the Nextflow ecosytem and speaking with pioneers in the field.</h4>
+          </div>
+        </div>
```
Comment (ewels) [inline templates/podcastsindex.ftl:17]:
```suggestion
        <div class="row">
          <div class="col-sm-4">
            <img src="/img/podcast_logo_horiz.svg" style="width:100%;">
          </div>
          <div class="col-sm-8">
            <h4 class='text-muted' style="margin-top: 7rem;">Talking about news in the Nextflow ecosytem and speaking with pioneers in the field.</h4>
          </div>
        </div>
```

**Review by mribeirodantas — CHANGES_REQUESTED** [review]
Cedric's episode image has an incorrect name below his avatar picture. Everything else is great!

**Review by mribeirodantas — APPROVED** [review]

**Comment by mribeirodantas** [comment]
I think it's good to merge. What do you think, @pditommaso ?

**Comment by pditommaso** [comment]
Wonder in the nav bar should be "Podcast" instead of "Podcasts" as it's "Blog" not Blogs" ?

**Comment by ewels** [comment]
> Wonder in the nav bar should be "Podcast" instead of "Podcasts" as it's "Blog" not Blogs" ?

Fine by me, I'll change it 👍🏻 

Note that the URL is `podcasts.html` but specific episodes are singular (eg. `podcast/2022/ep5_evan_floden.html`). This is because I couldn't figure out how to do an archive page for a custom post type any other way. But I don't think that anyone will notice or care about that, the nav bar is more important.

**Comment by pditommaso** [comment]
It was fast!

**Comment by pditommaso** [comment]
Great!

**Comment by ewels** [comment]
> It was fast!

Thanks to Lilia! Figma reusable components FTW - change in one place and all the graphics automatically update ✨

**Pull request merged**